File: scripts/etl/check_excel.js
const xlsx = require('xlsx');
const workbook = xlsx.readFile('NUEVA_DATA.xlsx');
const sheetName = workbook.SheetNames[0];
const worksheet = workbook.Sheets[sheetName];
const data = xlsx.utils.sheet_to_json(worksheet, { header: 1 });
console.log('Headers:', data[0]);
console.log('First row:', data[1]);

File: scripts/etl/check_excel.py
import openpyxl

wb = openpyxl.load_workbook('NUEVA_DATA.xlsx', data_only=True)
ws = wb.active
print("Headers:")
headers = [cell.value for cell in ws[1]]
print(headers)
print("First row:")
row2 = [cell.value for cell in ws[2]]
print(row2)
